core/src/mender-artifact.c

```diff
@@ -552,7 +552,8 @@ mender_artifact_read_manifest(mender_artifact_ctx_t *ctx) {
     return MENDER_DONE;
 }
 
-static mender_err_t
+/* TODO: Should we remove unused function? */
+__attribute__((unused)) static mender_err_t
 mender_create_provides_depends_node(const char *type, const char *value, mender_key_value_list_t **provides_depends) {
```

Comment:
@danielskinstad do you know if we plan to use this function, or should we remove it?

Reply:
We should remove it. I moved it to utils as `mender_utils_create_key_value_node`, but apparently forgot to remove it here.